packages/models/src/lib/plugin-config.ts

```diff
@@ -72,6 +72,16 @@ export const pluginConfigSchema = pluginMetaSchema
 
 export type PluginConfig = z.infer<typeof pluginConfigSchema>;
 
+export const pluginUrlsSchema = z
+  .union([z.url(), z.array(z.url()), z.record(z.url(), z.number().positive())])
```

Comment:
The weight schema should be the same as elsewhere. 0 should be valid - `.nonnegative()` instead of `.positive()`.

```suggestion
  .union([z.url(), z.array(z.url()), z.record(z.url(), weightSchema)])
```